client/my-sites/plan-features-2023-grid/index.tsx

```diff
@@ -998,14 +1033,22 @@ const ConnectedPlanFeatures2023Grid = connect(
 					isWpcomEnterpriseGridPlan( plan ) && planConstantObj.getPathSlug
 						? planConstantObj.getPathSlug()
 						: planObject?.product_name_short ?? '';
-				const storageOptions =
+				const storageFeatures =
 					( planConstantObj.get2023PricingGridSignupStorageOptions &&
 						planConstantObj.get2023PricingGridSignupStorageOptions(
 							showLegacyStorageFeature,
 							isCurrentPlan
 						) ) ||
 					[];
+				const storageOptions = storageFeatures.map( ( feature ) => {
+					const featureObject = getFeatureByKey( feature );
```

Comment:
Can we use `getPlanFeaturesObject` for `storageFeatures` like done above for `planFeatures` and `jpFeatures` (and avoid bringing in another local Calypso dependency here to access the objects)?

Reply:
Ah I assumed that since they're from the same module, it didn't matter. My mistake -- I'm happy to switch to an existing method 👍

Reply:
Addressed in [c0bf2c9](https://github.com/Automattic/wp-calypso/pull/79041/commits/c0bf2c962bf1621c404d54b8780ebb715ef01073)